test/vhswish-microkernel-tester.h

```diff
@@ -119,8 +119,8 @@ class VHSwishMicrokernelTester {
         const int32_t input_value = int32_t(uint32_t(input_zero_point() - input[i]) << 7);
         int32_t in = input_value * input_scale_div;
         in -= 16384;  // subtract 0.5 in Q15
-        in = std::min(in, 0);
-        in = std::max(in, -32768);
+        in = std::min<uint32_t>(in, 0);
+        in = std::max<uint32_t>(in, -32768);
```

Comment:
It is suspicious to be calling `std::max<uint32_t>` with a negative constant argument and a signed (`int32_t`) argument, is that correct?

Reply:
You are correct. I missed that. I think having int32_t make sense, and tried to recompile, successful. I will change this.